src/helpers/util.js

```diff
@@ -83,5 +83,47 @@ module.exports = {
   findLowerBoundValue: function (target, array) {
     var index = this.findLowerBound(target, array)
     return index >= 0 ? array[index] : null
+  },
+
+  findCall: findCall,
+  buildCallsPath: buildCallsPath
+}
+
+/**
+  * Find calls path from @args rootCall which leads to @args index (recursive)
+  *
+  * @param {Int} index - index of the vmtrace
+  * @param {Object} rootCall  - call tree, built by the trace analyser
+  * @return {Array} - return the calls path to @args index
+  */
+function buildCallsPath (index, rootCall) {
```

Comment:
I think you usually say "call path", even if there are multiple calls (also concerns documentation).